app/src/main/java/com/seay/game_of_thrones/model/GoTCharacter.java

```diff
@@ -0,0 +1,44 @@
+package com.seay.game_of_thrones.model;
+
+public class GoTCharacter {
+
+    private int id;
+    private String character;
+    private String actor;
+    private String description;
+    private String image;
+    private String caption;
+    private String credit;
+
+    public int getId() {
+        return id;
+    }
+
+    public String getCharacter() {
+        return character;
+    }
+
+    public String getActor() {
+        return actor;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public String getCaption() {
+        return caption;
+    }
+
+    public String getCredit() {
+        return credit;
+    }
+
+    public String getCharacterInformation(){
```

Comment:
We don't need this right now. This will live in another object later. The DTO is to be as dumb as possible.